lib/json/common.rb

```diff
@@ -175,7 +175,7 @@ def parse!(source, opts = {})
       :max_nesting  => false,
       :allow_nan    => true
     }.update(opts)
-    Parser.new(source, opts).parse
+    Parser.new(source, **(opts||{})).parse
```

Comment:
We don't need to explicitly do the `|| {}` as in the method params we are already doing it.